sympy/geometry/line.py

```diff
@@ -1280,6 +1280,57 @@ def distance(self, other):
             return S.Zero
         return self.perpendicular_segment(other).length
 
+    def nearest_points(l1, l2):
```

Comment:
`shortest_distance` or that matter, just `distance`?

Reply:
It's returning both the shortest distance and the points at the ends of that short segment. Perhaps we could return the segment and let the user take what they want, distance or args/pts. Then the function could be `shortest_segment_between`